dali/daliadmin/daadmin.cpp

```diff
@@ -158,19 +158,27 @@ void _export_(const char *path,const char *dst,bool safe)
 
 //==========================================================================================================
 
-void import(const char *path,const char *src,bool add)
+bool doImport(const char *path,const char *head,const char *tail,const char *xml,bool add,StringBuffer &out);
```

Comment:
any particular reason for forward declaration here, vs moving actual declaration to here before used?

Reply:
I did this because it is easy to review my changes. Since you just did the review, I may move the actual declaration to here in my next commit.